_rules/html-lang-valid-bf051a.md

```diff
@@ -56,46 +53,6 @@ The `lang` attribute specified is neither empty ("") nor only [whitespace](#whit
 <html lang="fr"></html>
 ```
 
-#### Passed Example 2
```

Comment:
There's only one passed example - I think it's worth adding some more passed examples from BCP-47:

de-AT
sr-Latn-RS
es-419
en-a-aaa-b-ccc-bbb-x-xyz
i-klingon

Without these it would be easy to write code that passes the ACT-R unit tests but flags valid BCP-47 language codes as invalid.

Reply:
We're not testing BCP 47 with these test cases though and I don't think we should.